sceval_frontend/src/components/AmChart.tsx

```diff
@@ -211,6 +211,7 @@ export const AmChart: React.FC<AmChartProps> = (props: AmChartProps) => {
         // nothing
       };
     }
+  // eslint-disable-next-line react-hooks/exhaustive-deps
   },        [sensorChart, props.zoomEventDispatchDelay, props.hasFocus]);
```

Comment:
We can't fix this missing dependencies warning here because it cause infinite loop so we will suppress the warning instead